source/Octodiff/Core/DeltaApplier.cs

```diff
@@ -1,12 +1,13 @@
 using System;
 using System.Collections;
 using System.IO;
-using Octodiff.Diagnostics;
 
 namespace Octodiff.Core
 {
     public class DeltaApplier
     {
+        private readonly int bufferLength = 4 * 1024 * 1024;
```

Comment:
Nit: use a `const` here.

Reply:
Thanks Adam, this has been addressed [here](https://github.com/OctopusDeploy/Octodiff/pull/39/commits/5b886de05dd2fa0ca1118cdefbc894ed6652e693).